tools/integrations/data/NavierStokesSolver.json

```diff
@@ -72,6 +63,11 @@
       "minimum": 0.01,
       "maximum": 1,
       "type": "number"
+    },
+    "_addLinearSolver": {
+      "title": "Add linear solver",
+      "type": "boolean",
+      "default": false
```

Comment:
will we have an option to use this 'displayed_name' for this internal flag?
Specifically here, I would prefer to have `Add linear solver config` as `adding linear solver` has different meaning